twitter-challenge.html

```diff
@@ -0,0 +1,115 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Twitter challenge</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <div class="grid">
+        <div class="rightsidebar">
+            <div class="Rcontainer">
+                <div class="searchbar">
+                    <div class="searchtext">
+                        <p>Search twitter</p>
+                    </div>
+
+
+                </div>
+                <div class="newsfeed">
+                    <p>Whats happening</p>
+                </div>
+                <div class="follow">
+                    <p>Who to follow</p>
+                </div>
+            </div>
+        </div>
+
+        <div class="main">
+            <div class="tweet-feed">
+                <div class="header">
+                    <p>home</p>
+                    
+                    <hr>
+
+                    <!-- <div class="whats-happening"> -->
+                    <div class="profile">
+                        <p class="profiletext">whats happening?</p>
+                    </div>
+                    <p class="tweetb">Tweet</p>
+                </div>
+            </div>
+            <div class="feed">
+                <div class="feed-grid">
+
+                    <div class="maintext">
+                        <div class="profile"></div>
+                        <p><b>Tom</b> @tomkennedy - 25m</p>
+
+                        <div class="images-undertext">
+                            <div class="icons" img scr="">1</div>
+                            <div class="icons">2</div>
+                            <div class="icons">3</div>
+                            <div class="icons">4</div>
+                        </div>
+
+
+                        <!-- </div> -->
+
+                    </div>
+                </div>
+
+            </div>
+        </div>
+
+        <div class="leftsidebar">
+            <div class="buttons">
```

Comment:
I would maybe say instead of class=nav, use a nav tag then instead of divs use buttons for correct semantics